packages/expo-dev-launcher/ios/Tests/EXDevLauncherPendingDeepLinkRegistryTest.swift

```diff
@@ -44,7 +44,7 @@ class EXDevLauncherPendingDeepLinkRegistryTest: QuickSpec {
 
       expect(registry.pendingDeepLink).to(beNil())
       expect(listener.lastDeepLink?.absoluteString).to(equal("http://localhost:1234"))
-      expect(listener.lastDeepLink).to(be(consumedURL))
+      expect((listener.lastDeepLink! as NSURL) === (consumedURL! as NSURL)).to(beTrue())
```

Comment:
Why this change? Having the assertion library do the equality check produces a clearer error message